rc/windowing/repl/x11.kak

```diff
@@ -15,7 +15,9 @@ All optional parameters are forwarded to the new window} \
            exit
         fi
         if [ $# -eq 0 ]; then cmd="${SHELL:-sh}"; else cmd="$@"; fi
-        setsid ${kak_opt_termcmd} ${cmd} -t kak_repl_window < /dev/null > /dev/null 2>&1 &
+        setsid ${kak_opt_termcmd} \
+               "xdotool getactivewindow set_window --name kak_repl_window \
+                && exec \"${cmd}\"" < /dev/null > /dev/null 2>&1 &
```

Comment:
How about instead of using xdotool, changing `${cmd}` to something like:

    "${SHELL:-sh}" -c "printf '\e]2;kak_repl_window\a'; \"${cmd}\""

Reply:
I didn'd know about this, TIL. This would be the same as `printf "$(tput tsl)kak_repl_window\a; \"${cmd}\""`, right? [This serverfault thread](https://serverfault.com/questions/23978/how-can-one-set-a-terminals-title-with-the-tput-command/23998#23998) would suggest, that your solution would rely on the terminal to support these capabilities. It worked for me on OpenBSD using the `st` terminal, though.

Im not sure which solution is more portable.

Reply:
The `tsl` terminfo string is supposed to take the cursor to a particular column of the status line. For example, `tput tsl 5` should take you to the sixth column, so any text you print would leave the first five columns undisturbed. Using `tsl` and `fsl` to represent the window title is common enough to be provided as the `xterm+sl` terminal component, but it's kind of a hack, so no official terminal descriptions use it - it's just there for end-users to use if they want.

That string has been the "set the window title string" sequence for decades, though, so I'd be surprised if there were any GUI terminal emulators that didn't support it. Even tmux supports it, these days.

Reply:
Alright, since the original command you suggested works even with my somewhat unusual `st` terminal, I'll believe you.

This solution seems cleaner then, since it doesn't rely on the window manager to activate the new window. I'll update the pull request.